donut/templates/contact.html

```diff
@@ -1,88 +1,104 @@
+		Lesley Nye
+		<br><br>
+		Beth Larranaga
+		<br><br>
+		Kim Quintana
+		</p>
+	    </div>
+	    <div class="triple-split-center medium-text">
+	    	<h4>Description</h4>
+		<p>
+		Dean of Undergraduate Students
+		<br><br>
+		Associate Dean of Students
+		<br><br>
+		Associate Dean of Undergraduate Students
+		<br><br>
+		Office Manager
+		<br><br>
+		Office Assistant
+		</p>
+	    </div>
+	    <div class="triple-split-right medium-text">
+		<h4>Email</h4>
+		<p><a href="mailto:jodabiri@caltech.edu" class="external text" title="mailto:jodabiri@caltech.edu" rel="nofollow">jodabiri@caltech.edu</a>
+		<br><br>
+		<a href="mailto:barbarag@caltech.edu" class="external text" title="mailto:barbarag@caltech.edu" rel="nofollow">barbarag@caltech.edu</a>
+		<br><br>
+		<a href="mailto:lnye@caltech.edu" class="external text" title="mailto:lnye@caltech.edu" rel="nofollow">lnye@caltech.edu</a>
+		<br><br>
+		<a href="mailto:rosel@caltech.edu" class="external text" title="mailto:rosel@caltech.edu" rel="nofollow">rosel@caltech.edu</a>
+		<br><br>
+		<a href="mailto:kquintan@caltech.edu" class="external text" title="mailto:kquintan@caltech.edu" rel="nofollow">kquintan@caltech.edu</a></p>
+	    </div>
+	 </div>
 
+
+	 <a name="Student_Life"></a><h3> <span class="mw-headline">Student Life</span></h3>
+	 <div>
+		 <div class="triple-split-left medium-text">
+			<h4>Name</h4>
+			<p>
+			Tom Mannion
+			<br><br>
+			Erik Snowberg
+			<br><br>
+			Peter Daily
+			<br><br>
+			Jon Webster
+			<br><br>
+			Denise Okamoto
+			<br><br>
+			Sue Chiachiaro</p>
+		</div>
+		<div class="triple-split-center medium-text">
+			<h4>Description</h4>
+			<p>
+			Senior Director, Student Activities and Programs
+			<br><br>
+			MOSH (Master of Student Houses)
+			<br><br>
+			Assistant VP, Student Affairs, Housing, Dining
+			<br><br>
+			Associate Director of Dining Services
+			<br><br>
+			Assistant Director of Events, Activities, and Programs
+			<br><br>
+			Assistant Director for Living-Learning Programs
+			</p>
+		</div>
+		<div class="triple-split-right medium-text">
+			<h4>Email</h4>
+			<p><a href="mailto:mannion@caltech.edu" class="external text" title="mailto:mannion@caltech.edu" rel="nofollow">mannion@caltech.edu</a>
+			<br><br>
+			<a href="mailto:snowberg@caltech.edu" class="external text" title="mailto:snowberg@caltech.edu" rel="nofollow">snowberg@caltech.edu</a>
+			<br><br>
+			<a href="mailto:pdaily@caltech.edu" class="external text" title="mailto:pdaily@caltech.edu" rel="nofollow">pdaily@caltech.edu</a>
+			<br><br>
+			<a href="mailto:jwebster@caltech.edu" class="external text" title="mailto:jwebster@caltech.edu" rel="nofollow">jwebster@caltech.edu</a>
+			<br><br>
+			<a href="mailto:denise@caltech.edu" class="external text" title="mailto:denise@caltech.edu" rel="nofollow">denise@caltech.edu</a>
+			<br><br>
+			<a href="mailto:suec@caltech.edu" class="external text" title="mailto:suec@caltech.edu" rel="nofollow">suec@caltech.edu</a></p>
+		</div>
```

Comment:
Would be nice if all this people/title/email HTML could be generated from a list of administrators in some Python/JSON file